src/hooks/useHtmlPaste/index.ts

```diff
@@ -0,0 +1,129 @@
+import {useNavigation} from '@react-navigation/native';
+import ExpensiMark from 'expensify-common/lib/ExpensiMark';
+import {useCallback, useEffect} from 'react';
+import type UseHtmlPaste from './types';
+
+const useHtmlPaste: UseHtmlPaste = (textInputRef, checkComposerVisibility, isUnmountedOnBlur = true) => {
+    const navigation = useNavigation();
+
+    /**
+     * Set pasted text to clipboard
+     * @param {String} text
+     */
+    const paste = useCallback((text: string) => {
+        try {
+            document.execCommand('insertText', false, text);
+            // Pointer will go out of sight when a large paragraph is pasted on the web. Refocusing the input keeps the cursor in view.
+            textInputRef.current?.blur();
+            textInputRef.current?.focus();
+            // eslint-disable-next-line no-empty
+        } catch (e) {}
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can you add a comment explaining why we have an empty catch here? Ideally, we should not be using `eslint-disable`

Reply:
It was totally copy pasted from it has no explanation
https://github.com/Expensify/App/blob/6e2c58de92b81c754390a2dd9fc45e6a77557974/src/components/Composer/index.tsx#L163-L164
I guess its just there for safety.

Reply:
Yeah I saw, I hoped maybe you knew... this is a good demonstration of why confusing / non straightforward code without any comment is bad 😬

Reply:
This can be addressed by adding a comment. Please add something @FitseTLT. By passing the empty array we will ensure that the callback will only be created once.